packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -103,6 +110,226 @@ const en = {
     subtext: 'Restart the application after running the script',
     checkbox: 'I have already ran the command',
   },
+  initAccount: {
```

Comment:
Aggregate all `Add Account` related properties into a single object called `addAccountDialog`

Reply:
this was not done by me